src/DependencyInjection/Configuration.php

```diff
@@ -164,13 +164,17 @@ private function getOptionsConfig(): ArrayNodeDefinition
                 ->scalarNode('security_cover_url')
                     ->defaultValue('https://placehold.co/1000')
                 ->end()
-                ->scalarNode('theme_base')
-                    ->defaultValue('slate')
+                ->enumNode('theme_base')
+                    ->values(['slate', 'gray', 'zinc', 'neutral', 'stone'])
+                    ->defaultValue('gray')
```

Comment:
I used `slate`, because it is the one that is closest to the current blue.
But we can also use the default values from Tabler.